lametro/templates/base.html

```diff
@@ -83,7 +83,7 @@
 
         {% get_alerts as alerts %}
         {% if alerts %}
-            {% for alert in alerts|dictsort:"pk" %}
+            {% for alert in alerts %}
```

Comment:
Remove the previous sort now that they're always sorted chronologically.

Reply:
Great call to move the ordering up into the model so that it's consistent anywhere we're displaying the alerts!